README.adoc

```diff
@@ -193,6 +299,50 @@ It defines whether the buidler method can have a prefix or not. By default the b
 - enumMappingMethod (default `name`)
 When an enum type is defined as a field, we need to define what type of representation is coded in the map.
 
+== Annotations
+
+There are annotations which helps to configure the mapping between Map and the interface.
+The annotations have to be defined in the getter method.
+
+=== @Key
+
+This annotation is used to define the key name in map for the related field.
+
+=== @Embedded
+
+Sometimes in the Map storing flat information which is better to shown as an separated
```

Comment:
My suggestion:

"Sometimes we need to store information in a hierarchical structure with or without dedicated interfaces. For such cases we can use the `@Embedded` annotation."